thirdparty_containers/README.md

```diff
@@ -1,5 +1,23 @@
+# Third Party Container Tests
+### Basic Steps:
+- Learn how to run the application tests that you intend to automate in the build manually first, and find out any special dependencies the application testing may have.
+- Clone https://github.com/AdoptOpenJDK/openjdk-tests.git and look at thirdparty_containers directory.
+- Copy the 'example-test' subdirectory and rename it after the application you are adding. 
+- Modify the files in your new sub-directory according to your needs. 
+- Check in the changes into https://github.com/[YOUR-BRANCH]/openjdk-tests and test it using a personal build. 
```

Comment:
You could hyperlink "test it using a personal build" with this link: https://github.com/AdoptOpenJDK/openjdk-tests/wiki/How-to-Run-a-Personal-Test-Build-on-Jenkins